Calculator/Calculator/Model/ExpressionParser.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  ExpressionParser.swift
+//  Calculator
+//
+//  Created by 비모 on 2023/06/06.
+//
+
+enum ExpressionParser {
+    static func parse(from input: String) -> Formula {
+        let numbers = componentsByOperators(from: input).compactMap {
+            Double($0)
+        }
+        
+        var operands = CalculatorItemQueue<Double>()
+        var operators = CalculatorItemQueue<Operator>()
+        
+        numbers.forEach {
+            operands.enqueue($0)
+        }
```

Comment:
개인적인 견해지만, 코드 구현 순서가 조금 어색하다는 생각이 들어요!😀
numbers를 선언하고 연산하는 중간에 operands, operators 변수가 있는게 가독성을 헤치는 것 같습니다.
떄문에 아래와 같은 코드는 어떨까요?
```suggestion
        var operands = CalculatorItemQueue<Double>()
        var operators = CalculatorItemQueue<Operator>()
        
        componentsByOperators(from: input).compactMap {
            Double($0)
        }
        .forEach {
            operands.enqueue($0)
        }
```

Reply:
제안 주신 코드가 제가 지향하는 코드 스타일이네요!
아래 선언한 `componentsByOperators` 메서드를 불러 쓰는 과정에 집중하다보니 놓치고 지나간 부분이네요 👀
코멘트 감사합니다!